### DISCUSSION TITLE
How to use in the table body slot and multiple selection

### DISCUSSION THREAD

**Discussion by Goodwin-gls** [quasarframework/quasar#15875] [General - Components / Directives / etc]
I think the body slot removes the selector slot so the selectors don't show up in the table. Selecting all rows is also slow. how to solve this problem without custom first column with select ?

**Comment by ssuess** [comment]
I have the same problem, if  `<template v-slot:body="props">` is defined, using `selection="multiple"` will add an item in the header of my table, but will not put a checkbox at the beginning of each row (as it will when the template for body is not defined. Is there some example code somewhere of how to add back these checkboxes to the first cell?

**Reply by metalsadman** [reply]
it will not add automatically since when using that slot (for whole columns), the user is in-charged of adding it. https://codepen.io/metalsadman/pen/LYpOpdX

**Reply by ssuess** [reply]
Thanks!
